aws-opentelemetry-distro/src/amazon/opentelemetry/distro/llo_handler.py

```diff
@@ -137,6 +138,35 @@ class PatternConfig(TypedDict, total=False):
         "role": ROLE_USER,
         "source": "prompt",
     },
+    # OTel GenAI Semantic Convention used by the latest Strands SDK
+    # References:
+    # - OTel GenAI SemConv: https://opentelemetry.io/docs/specs/semconv/gen-ai/gen-ai-events/
+    # - Strands SDK PR(introduced in v0.1.9): https://github.com/strands-agents/sdk-python/pull/319
```

Comment:
The strands sdk for python is at v1.5.0: https://github.com/strands-agents/sdk-python/releases
Have we audited other breaking changes and confirmed this is the only update needed?

Reply:
Let's follow this with Strands SDK team. This fix is to fix the known issue reproted.